lib/lib.ml

```diff
@@ -68,7 +67,7 @@ let validate_nested_protocols (ast : scr_module) ~verbose =
   let ast = Protocol.rename_nested_protocols ast in
   show ~f:show_scr_module ~sep:"\n---------\n\n" ast ;
   let global_t = Gtype.global_t_of_module ast in
-  let global_t = Gtype.normalise_global_t global_t in
+  (* let global_t = Gtype.normalise_global_t global_t in *)
```

Comment:
same, do we need to keep this?

Reply:
https://github.com/nuscr/nuscr/issues/130